docs/cookbook/recipe_custom_view.rst

```diff
@@ -0,0 +1,134 @@
+Creating a Custom Admin View
+============================
+
+Sometimes you may want to create an additional view in your SonataAdmin
+but you do not have or need an Entity to connect it to. You also probably
+want to add it to sidebar with the rest of your admin pages? Here we will
+present a full working example of how is that achievable.
+
+The recipe
+----------
+
+SonataAdmin provides a very straightforward way of adding your own custom
+view.
+
+To do this we need to:
+
+- create an admin class for your custom view
+- extend the ``SonataAdmin:CRUD`` Controller and tell our admin class to
+  use it
+- create a template that will represent your custom view
+
+Create an admin class
+^^^^^^^^^^^^^^^^^^^^^
+
+First we need to create an admin class and we will clear all routes except
+``list``.
+
+.. code-block:: php
```

Comment:
From https://symfony.com/doc/current/contributing/documentation/standards.html :

> The :: shorthand is preferred over .. code-block:: php to begin a PHP code block (read the Sphinx documentation to see when you should use the shorthand);

Reply:
Did not know this, in docs it is mostly ` .. code-block:: php`

Reply:
Yeah most people don't know about this rule, but it exists :P